tests/misc.cpp

```diff
@@ -1302,6 +1306,12 @@ get_solver_str(QudaInverterType type)
   case QUDA_CA_CG_INVERTER:
     ret = "ca-cg";
     break;
+  case QUDA_CA_CGNE_INVERTER:
+    ret = "ca-cgne";
+    break;
+  case QUDA_CA_CGNR_INVERTER:
+    ret = "ca-cgnr";
+    break;
```

Comment:
clang-format suggested the following change:


```
-  case QUDA_CA_CGNE_INVERTER:
-    ret = "ca-cgne";
-    break;
-  case QUDA_CA_CGNR_INVERTER:
-    ret = "ca-cgnr";
-    break;
+  case QUDA_CA_CGNE_INVERTER: ret = "ca-cgne"; break;
+  case QUDA_CA_CGNR_INVERTER: ret = "ca-cgnr"; break;
```

<!-- identifier: clang-format -->
<!-- File: tests/misc.cpp -->